documentation/source/install/on_premise_solutions.rst

```diff
@@ -285,17 +285,17 @@ Run Sample Applications on the OpenEBS Setup
 
       kubectl describe pod <pod name>
 
-* The storage volume that is the persistent volume associated with the persistent volume claim, can be viewed using the *vsm-list* command in the maya-apiserver pod.
+* The storage volume that is the persistent volume associated with the persistent volume claim, can be viewed using the *volume list* command in the maya-apiserver pod.
   :: 
 
-      name@MayaMaster:~$ kubectl exec maya-apiserver-1633167387-v4sf1 -c maya-apiserver -- maya vsm-list
+      name@MayaMaster:~$ kubectl exec maya-apiserver-1633167387-v4sf1 -c maya-apiserver -- maya volume list
       Name                                      Status
       pvc-a2a6d71f-5b21-11e7-bf1c-000c298ff5fc  Running
      
-* Verify that the storage volume is receiving input/output by checking the increments to *DataUpdateIndex* in the output of the vsm-stats command issued in the maya-apiserver pod. Some additional performance statistics are also available in the command output.
+* Verify that the storage volume is receiving input/output by checking the increments to *DataUpdateIndex* in the output of the volume stats command issued in the maya-apiserver pod. Some additional performance statistics are also available in the command output.
```

Comment:
can you please put shell commands in a shell coats (``) i.e `volume stats`

Reply:
I noticed that commands are wrapped in asterisks (**) instead of the normal shell coats. Should they be wrapped in asterisks for consistency with the rest of the documentation, or in shell coats for readability?

Reply:
I think it will be better to have, these commands in shell coats.